book/website/ethical-research/self-reflection/sr-power.md

```diff
@@ -0,0 +1,27 @@
+(er-self-reflection-power-privilege)= 
+# Power and privilege
+
+When thinking about positionality, it is important to understand how we might hold power and privilege relative to the people we work with, those whose data we are studying, or the problems we are attempting to solve. 
+Our experiences of power and privilege or oppression are tightly interwoven with our experiences of class, race, and gender and more {cite:ps}`crenshaw1989demarginalizing, collins1990black`. 
+These are important to consider because themes of power and privilege are present in all of our contexts. 
+Having privilege can make us unaware of related issues that do not directly affect us.
+
+To illustrate, here are some examples of how power and privilege present as structural issues in data science:  
+<!-- Welcome more examples here! -->
+- Racial injustice and patriarchal systems have meant that data science has historically been dominated by white men. This means that there has not been a diverse group of people to direct the aims of data science work, and so many groups are likely to be overlooked when considering the negative impacts and potential benefits of data science.
+- The English language is the default in many programming languages and academic settings, which creates barries to knowledge sharing and inclusion. Rather than English speakers learning other languages, it is generally expected that non-English speakers learn English in order to participate and collaborate, which frames English as the default. 
```

Comment:
```suggestion
- **The English language is the default** in many programming languages and academic settings, which creates barries to knowledge sharing and inclusion. Rather than English speakers learning other languages, it is generally expected that non-English speakers learn English in order to participate and collaborate, which frames English as the default. 
```